ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.19__amd64-nightly.yaml

```diff
@@ -1528,6 +1528,18 @@ tests:
     test:
     - chain: openshift-e2e-test-qe
     workflow: cucushift-installer-rehearse-azure-ipi-disconnected-cco-manual-workload-identity
+- as: azure-ipi-external-oidc-entraid-tp-f14
+  cron: 17 19 1,15 * *
+  steps:
+    cluster_profile: azure-qe
+    env:
+      BASE_DOMAIN: qe.azure.devcluster.openshift.com
+      FEATURE_SET: TechPreviewNoUpgrade
+    test:
+    - chain: idp-external-oidc-entraid
+    - ref: openshift-extended-test
+    - ref: openshift-e2e-test-qe-report
+    workflow: cucushift-installer-rehearse-azure-ipi
```

Comment:
@xiuwang , corresponding to step `...-external-oidc-console-app-redirect-uri-add`, you would need to add [...-external-oidc-console-app-redirect-uri-remove](https://steps.ci.openshift.org/reference/cucushift-hypershift-extended-external-oidc-console-app-redirect-uri-remove) here:
```
    test:
    - chain: idp-external-oidc-entraid
    - ref: openshift-extended-test
    - ref: openshift-e2e-test-qe-report
    post:
    - ref: ...-external-oidc-console-app-redirect-uri-remove
    workflow: cucushift-installer-rehearse-azure-ipi
```